### PR TITLE
Ecf dashboard breadcrumbs

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/helpers/think_feel_do_dashboard/application_helper.rb [hunk 1]
```diff
@@ -2,5 +2,43 @@ module ThinkFeelDoDashboard
...
+                              )
+                  )
+          end
+        end
+      )
+    end
+
+    private
+
+    def show_breadcrumbs?
```
Comment (gwensmu) [inline app/helpers/think_feel_do_dashboard/application_helper.rb]: this is nifty. i'm learning right now.

Comment (Tag0Mag0) [inline app/helpers/think_feel_do_dashboard/application_helper.rb]: Is this the preferred Rubocop indentation? I can never tell what it wants from me.

**Comment by ericcf** [comment]
Not sure if it's the prettiest indentation, but it passes...

**Comment by ericcf** [comment]
@Tag0Mag0 @gwensmu would one of you mind merging this assuming you have no objections?

**Pull request merged**